demo/python/basic/secure-xgboost-demo.py

```diff
@@ -6,7 +6,7 @@
 OE_ENCLAVE_FLAG_SIMULATE = 2
 
 print("Creating enclave")
-HOME_DIR = os.getcwd() + "/../../../"
+HOME_DIR = os.getcwd() + "/"
```

Comment:
Can you revert this and the other path change on line 54 back to what it was originally? Good catch on your part though.

We realized that it's better to not use `os.getcwd()` here but instead use `os.path.dirname(os.path.realpath(__file__))` to get the full path to the file. See https://stackoverflow.com/questions/5137497/find-current-directory-and-files-directory for more info.

We'll make the corresponding fixes in another PR.